consent-ws/src/main/java/org/broadinstitute/consent/http/resources/DataRequestReportsResource.java

```diff
@@ -79,4 +89,34 @@ public Response downloadReviewedDARs() {
             return createExceptionResponse(e);
         }
     }
+
+    private Boolean requiresManualReview(Document dar) throws IOException {
+        Map<String, Object> form = parseAsMap(dar.toJson());
+        for (String field: fieldsForManualReview) {
+            if (form.containsKey(field)) {
+                if (Boolean.valueOf(form.get(field).toString())) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
+    private Map<String, Object> parseAsMap(String str) throws IOException {
+        ObjectReader reader = mapper.readerFor(Map.class);
+        return reader.readValue(str);
+    }
+
+    private String[] fieldsForManualReview = {
```

Comment:
This is the same in `DataAccessRequestResource`. We should refactor both into a single array